kafka/pykafka/broker.py

```diff
@@ -16,7 +16,8 @@
 
 class Broker(base.BaseBroker):
 
-    def __init__(self, id_, host, port, handler, timeout):
+    def __init__(self, id_, host, port, handler, timeout,
+                 buffer_size=64 * 1024):
```

Comment:
This is a total formatting thing, so feel free to ignore, but when a parameter list has to go on two lines, I've found this style to be cleaner:

```python
def __init__(self,
             id_,
             host,
             port,
             handler,
             timeout,
             buffer_size=64 * 1024):
    """docstring blah blah blah"""
````

The logic in my head, which again ignore if you disagree since this is highly subjective, is that if it's one line you can scan it quickly. You can't really scan two lines as easily when reading. By having one on each line, you lose the ability to scan, which was lost already, but gain the ability to read because it's not a huge blob of text.  It also makes adding/removing parameters easier, especially if they're in the middle of the argument list.

I've seen this done both ways, but adding a couple more params to this feels very unwieldy to me:
```
def __init__(self, id_, host, port, handler, timeout,
             buffer_size=64 * 1024, socket_timeout_ms=1000 * 30,
             required_acks=4):